bin/testEnvRootMinikube.sh

```diff
@@ -48,6 +48,31 @@ function startMinikubeNone() {
     export MINIKUBE_WANTREPORTERRORPROMPT=false
     export MINIKUBE_HOME=$HOME
     export CHANGE_MINIKUBE_NONE_USER=true
+
+    # Troubleshoot problem with Docker build on some CircleCI machines
+    if [ -f /proc/sys/net/ipv4/ip_forward ]; then
+      echo "IP forwarding setting: $(cat /proc/sys/net/ipv4/ip_forward)"
+      echo "My hostname is:"
+      hostname
+      echo "My distro is:"
+      cat /etc/*-release
+      echo "Contents of /etc/sysctl.d/"
+      ls -l /etc/sysctl.d/ || true
+      echo "Contents of /etc/sysctl.conf"
+      grep ip_forward /etc/sysctl.conf
+      echo "Config files setting ip_forward"
+      find /etc/sysctl.d/ -type f -exec grep ip_forward \{\} \; -print
+      if [ "$(cat /proc/sys/net/ipv4/ip_forward)" -eq 0 ]; then
```

Comment:
For this to work - `-eq` expects 0 to be quoted `"0"`.  See https://stackoverflow.com/questions/20449543/bash-equality-operators-eq for more details.